portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,31 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private final ArrayList<String> comments = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello stranger! Welcome to my portfolio");
+    // Dummy comments for list
```

Comment:
nit: Include a period at the end of comments. (Here and elsewhere in this file)

```suggestion
    // Dummy comments for list.
```

Reply:
Done!